Velero installation guide and overrides docs (#4909)
kyma-project/kyma: docs/kyma/04-01-overview.md

| `service-catalog-addons` | ✅ | ✅ |
| `nats-streaming` | ✅ | ✅ |

> > \* In order to install velero, it needs to be set up first. Please follow the Kyma Backups setup instructions below.

Reviewer: I will confirm this part on Monday. 
(the best way to put it).

> > \* In order to install velero, it needs to be set up first. Please follow the Kyma Backups setup instructions below.

Reviewer: ```suggestion
> \* To install Velero, you need to set it up first. To do so, use the Kyma backup [setup instructions](/components/backup#overview-overview).
```

Author: Fixed thanks!

## Installation guides

Follow these installation guides to install Kyma locally or on a cluster:

- [Install Kyma locally](#installation-install-kyma-locally)
- [Install Kyma on a cluster](#installation-install-kyma-on-a-cluster)

Read the rest of the installation documents to learn how to:
- [Disable the selected components' installation or install them separately](#configuration-custom-component-installation)
- [Upgrade Kyma to a new version](#installation-upgrade-kyma)
- [Update Kyma](#installation-update-kyma)
- [Reinstall Kyma](#installation-reinstall-kyma)

> - [Setup Kyma Backups](/components/backup)

Reviewer: ```suggestion
- [Set up Kyma backup](/components/backup#overview-overview)
```

Author: I do not think this is gramatically correct...
Either we say "Set up Kyma Backups" in plural so we do not need an article, or we say "Set up the kyma backup".
The current suggestion sounds strange to me .

Author: Fixed thanks!
